gc/base/standard/ConcurrentGC.cpp

```diff
@@ -487,6 +487,36 @@ MM_ConcurrentGC::reportConcurrentRememberedSetScanEnd(MM_EnvironmentBase *env, u
 	);
 }
 
+void MM_ConcurrentGC::preConcurrentInitializeStatsAndReport(MM_EnvironmentBase *env)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
+	_concurrentPhaseStats._cycleID = _concurrentCycleState._verboseContextID;
+	_concurrentPhaseStats._startTime = omrtime_hires_clock();
+	TRIGGER_J9HOOK_MM_PRIVATE_CONCURRENT_PHASE_START(
+			_extensions->privateHookInterface,
+			env->getOmrVMThread(),
+			omrtime_hires_clock(),
+			J9HOOK_MM_PRIVATE_CONCURRENT_PHASE_START,
+			&_concurrentPhaseStats);
+}
+
+void MM_ConcurrentGC::postConcurrentUpdateStatsAndReport(MM_EnvironmentBase *env)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
+	_concurrentPhaseStats._cycleID = _concurrentCycleState._verboseContextID;
+	_concurrentPhaseStats._cardTableStats = _cardTable->getCardTableStats();
+	_concurrentPhaseStats._collectionStats = _stats;
+
+	Assert_MM_true(0 != _concurrentPhaseStats._endTime);
```

Comment:
Check the `endTime` here. But could only check if the `endTime` is set, no guarantee it is the correct value.